modulemd/modulemd-yaml-util.c

```diff
@@ -504,6 +504,17 @@ modulemd_yaml_parse_string_set (yaml_parser_t *parser, GError **error)
       yaml_event_delete (&event);
     }
 
+  /* Work around false-positive in clang static analysis which thinks it's
+	 * possible for this function to return NULL and not set error.
+	 */
```

Comment:
Indentation is still weird--caused by a hard tab at the beginning of those two lines.

Reply:
Thanks, it looked fine on my editor because the tabs were four spaces wide. Fixed now.

Reply:
No problem. I thought `clang-format` expanded hard tabs...

Reply:
It doesn't touch tabs that are inside a comment block.